src/webgpu/api/validation/encoding/beginRenderPass.spec.ts

```diff
@@ -15,6 +15,8 @@ Notes:
 >     - {all possible {depth, stencil} load ops, load values {in range, negative, too large}}
 >     - all possible {depth, stencil} store ops
 >     - depthReadOnly {t,f}, stencilReadOnly {t,f}
+> - occlusion query set
+>     - tested in queries/general.spec.ts
```

Comment:
This whole notes section is part of the TODO. This should go outside of the TODO (it can go above) like "Note: render pass 'occlusionQuerySet' validation is tested in queries/general.spec.ts"